src/commands/deploy/latest.js

```diff
@@ -98,7 +98,7 @@ const printDeploymentStatus = async (
           }
         }
       } else {
-        output.ready(`Aliases assigned ${deployStamp()}`);
+        output.ready(`Deployment complete ${deployStamp()}`);
```

Comment:
What about the mentions of alias above? It looks like this is only printed when `aliasList.length > 1`?

Reply:
Also, line 127 shows "Deployment ready" instead of complete. Is there a difference?

Reply:
Updated, let's have @leo sign off on the changes.